tools/storybook/package.json

```diff
@@ -11,8 +11,10 @@
 	"author": "Automattic",
 	"license": "GPL-3.0-or-later",
 	"scripts": {
-		"build-storybook": "pnpm build-woocommerce && ./import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true storybook build -c ./.storybook",
+		"build-storybook": "pnpm build-woocommerce && pnpm build-woocommerce-blocks-storybook && ./import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true storybook build -c ./.storybook",
+		"postbuild-storybook": "cp ../../plugins/woocommerce-blocks/storybook/dist/iframe.html ./storybook-static/assets/woocommerce-blocks",
```

Comment:
This is a bit annoying, but without the `iframe.html` the stories wouldn't load. See: https://github.com/storybookjs/storybook/issues/24978
And the `staticDirs` config skips this on purpose. An alternative is that we remove the use of `staticDirs` and just copy the entire `dist` directory over as part of this command.